physical-based-animations.go

```diff
@@ -48,6 +48,12 @@ func updateParticles(
 					math.Atan2(unitNormalVector.Y, unitNormalVector.X)))
 
 			particles[i].speed = newSpeed.Scaled(coefficientOfRestitution)
+
+			if positionIntegrator == Verlet {
+				particles[i].nextPosition = newPosition.Add(particles[i].speed.Scaled(PixelsPerMeter).Scaled(dt)).Add(
```

Comment:
As I mentionned in previus MR, wrapping lines after 100 charaters is good habit. As you can see this line is not completely visible, I have to scroll:
<img width="1015" alt="screen shot 2018-01-15 at 20 55 47" src="https://user-images.githubusercontent.com/22389946/34959361-801e2ea0-fa36-11e7-9921-1413f6a5a832.png">
What do you think about line spliting in go lang ? Is this suggestion irelevant?

Reply:
Github autobreaks the line for me and this math is more readable in a line of code rather then being split imo
![image](https://user-images.githubusercontent.com/17459600/34959840-8c0e4310-fa38-11e7-9a17-5fe02fa3247f.png)